### DISCUSSION TITLE
some suggestions!

### DISCUSSION THREAD

**Discussion by Quikkby** [keiwando/toonsquid#19] [Ideas]
hello, i enjoy the program alot! and have even showed my friends it (one even moved onto toonsquid) but the only problem is its lacking in some important features other programs have! and ill show you which ideas that may be helping with overall animating satisfaction

1. a bucket tool that has an option to refer to above layers or  a specific layer, sometimes people would rather use this so they can later clipping mask and color the layer nicley, along with that, a gradient option too would be nice! and another option is to close gaps. or predict gaps

2. a higher smoothing option (i have unbeleivable shaky hands) 

3. filter options, border effects, glowing effects, and generated noise.. only some come to mind

4. last but not least, more pixel brushes (like in flipnote hatena) where it has more patterns that look darker or lighter

thank you so much if you do atleast implement some of them <3

**Comment by keiwando** [comment]
Thank you very much for the feedback! Some of these features are already planned and I've noted down the other ones as feature requests.

**Comment by keiwando** [comment]
Higher brush smoothing options are now possible with the 1.6 update
